parquet-column/src/main/java/org/apache/parquet/column/Comparators.java

```diff
@@ -0,0 +1,273 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.parquet.column;
+
+import org.apache.parquet.io.api.Binary;
+import org.apache.parquet.schema.OriginalType;
+import org.apache.parquet.schema.PrimitiveType;
+
+import java.util.Comparator;
+
+/**
+ * Utility class to provide {@link java.util.Comparator} implementations for logical types.
+ */
+public class Comparators {
+
+  public static class BooleanComparator implements Comparator<Boolean> {
+    @Override
+    public int compare(Boolean o1, Boolean o2) {
+      return compare(o1.booleanValue(), o2.booleanValue());
+    }
+
+    public int compare(boolean b1, boolean b2) {
+      return Boolean.compare(b1, b2);
+    }
+
+    /**
+     * Returns the string representation of the specified value for debugging/logging purposes
+     */
+    public String toString(boolean b) {
```

Comment:
I know that it is convenient to add `toString` here, but I don't think it makes sense with the `Comparator` API. I think it would be better to pass `Type` to `Statistics`, then use that type to get a comparator and to do the coercion to string.